synapse_data_warehouse/synapse_event/dynamic_tables/V2.56.0__add_filter_to_filedownload_dynamic_table.sql

```diff
@@ -0,0 +1,48 @@
+-- Introduce the dynamic table
+USE SCHEMA {{database_name}}.SYNAPSE_EVENT; --noqa: JJ01,PRS,TMP
+
+CREATE OR REPLACE DYNAMIC TABLE FILEDOWNLOAD
+    (
+        TIMESTAMP TIMESTAMP_NTZ(9) COMMENT 'The time when the file download event is pushed to the queue for recording, after generating the pre-signed url.',
+	    USER_ID NUMBER(38,0) COMMENT 'The id of the user who downloaded the file.',
+	    PROJECT_ID NUMBER(38,0) COMMENT 'The unique identifier of the project where the downloaded entity resides. Applicable only for FileEntity and TableEntity.',
+	    FILE_HANDLE_ID NUMBER(38,0) COMMENT 'The unique identifier of the file handle.',
+	    DOWNLOADED_FILE_HANDLE_ID NUMBER(38,0) COMMENT 'The unique identifier of the zip file handle containing the downloaded file when the download is requested as zip/package, otherwise the id of the file handle itself.',
+	    ASSOCIATION_OBJECT_ID NUMBER(38,0) COMMENT 'The unique identifier of the Synapse object (without ''syn'' prefix) that wraps the file.',
+	    ASSOCIATION_OBJECT_TYPE VARCHAR(16777216) COMMENT 'The type of the Synapse object that wraps the file, e.g., FileEntity, TableEntity, WikiAttachment, WikiMarkdown, UserProfileAttachment, MessageAttachment, TeamAttachment.',
```

Comment:
Can we update the column comments to describe which columns are part of the composite primary key (i.e. the columns used for de-duplication)?

[I use this as an example](https://github.com/Sage-Bionetworks/snowflake/pull/205/files#diff-f3ba8580d8def541b7b89ff382b2fe82fa8089b9702f06eeaafffa4a2696ee46)